### PR TITLE
refactor: add missing tools and config

### PR DESCRIPTION
**Pull request by palafoxernesto**
### Description

Adds .github/development workflow and mypy configuration

### How was it tested?
- [x] unit

### Checklist

- [x] Does your code follow the project style guide?
- [x] Have you added tests that cover the changes and run them?

### CODE REVIEW

**Review by manicar2093 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -31,3 +39,6 @@ codeclimate:
 
 commit_check:
 	cz check --rev-range origin/master..HEAD
+
+bump:
+	cz bump --changelog --yes
```
Comment (daninfpj) [inline Makefile:44]: EOL 👁️

File: setup.cfg [hunk 2]
```diff
@@ -73,3 +74,4 @@ omit=
     src/__init__.py
     src/*/handler.py
     src/*/__init__.py
+
```
Comment (daninfpj) [inline setup.cfg:77]: Estaría bueno tener acá la conf básica de `mypy` que usamos.

**Review by daninfpj — APPROVED** [review]
🎖️

**Review by Gonzasestopal — APPROVED** [review]

**Review by sairoko12 — APPROVED** [review]
🚢

**Pull request merged**